baresip/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG IMAGE=ghcr.io/baresip/docker/librem:latest
 FROM ${IMAGE}
 ARG VERSION
 ARG RELEASE
-RUN install_packages libopus-dev libasound2-dev libmosquitto-dev libspandsp-dev
+RUN install_packages libopus-dev libasound2-dev libmosquitto-dev libspandsp-dev libssl-dev libpulse-dev
```

Comment:
libssl-dev should not be needed since imported from rem/re image.

Reply:
The install_packages will skip whatever is installed already, no? The base image might change and then we'll need to change this again if we remove it.

Reply:
I prefer to keep it clean and add only baresip related dependencies here, we rely for baresip build on the other dist imports and dev packages already like `clang-11 make cmake pkg-config git libssl-dev wget ca-certificates libz-dev`.

Reply:
OK, cool. One other idea then it would be to have a separate 'builder' image for only the modules, then copy them to the final image.